src/agent/application.cpp

```diff
@@ -255,4 +199,89 @@ void Application::HandleSignal(int aSignal)
     signal(aSignal, SIG_DFL);
 }
 
+void Application::ConstructRcpMode(const std::string &aRestListenAddress, int aRestListenPort)
```

Comment:
I'm concerned that there may be uninitialized members after some time, since the compiler doesn't throw an error if somebody forgets to initialize a member.

Reply:
As discussed offline, we still try to make these members be initialized in the initliazer list as possible in the future when other components can work under RCP mode.